### PR TITLE
add support for pnpm

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

**Comment by FabianLars** [comment]
Why didn't that run fail in the other PR too lol

**Comment by FabianLars** [comment]
`tauri init` will fail for npm and pnpm, because we don't add a script in package.json for it. it would work with pnpm if we remove the "run" from `pnpm run tauri init`, but then we'd still have npm, so adding a script is the way to go here. ~~Probably will do this tomorrow tho.~~

File: package.json [hunk 1]
```diff
@@ -108,6 +108,7 @@
   },
   "dependencies": {
     "glob": "7.1.7",
+    "jsonfile": "^6.1.0",
```
Comment (amrbashir) [inline package.json]: could be replaced with node built-in `JSON` and `fs` module

Comment (FabianLars) [reply package.json]: Yeah, you're right. 1:00am me wasn't able to make it work lol

**Comment by lucasfernog** [comment]
Change file here is wrong. Must use `"` instead of `'` on the package name.

**Review by lucasfernog — APPROVED** [review]

**Comment by FabianLars** [comment]
vs code's autoformat-on-save is not my friend today...

**Comment by FabianLars** [comment]
@lucasfernog should i split it into 2 change files? because the changes are somewhat unrelated to each other.
Btw, no idea why covector and the "Files changed"-tab are still crying about the changefile :/

**Comment by lucasfernog** [comment]
Github somehow thinks your change file is a binary file. Maybe delete that one and recreate lol

**Comment by lucasfernog** [comment]
And yeah I'd split into two change files.

**Comment by lucasfernog** [comment]
Finally lol

**Pull request merged**